app/api/deleteDocument/route.ts

```diff
@@ -0,0 +1,96 @@
+import { NextResponse } from 'next/server';
+import prisma from '@/utils/prisma';
+import { getAuth } from '@clerk/nextjs/server';
+
+interface DeleteFileParams {
+  accountId: string;
+  apiKey: string;
```

Comment:
Does this need to be a parameter?

Reply:
I followed the docs of bytescale to implement functionality so I just added this to resolve typescript errors. The Api key is taken from the env variables and then passed to this. Can be removed but i though of keeping the same syntax as docs.